layouts/release/section.html

```diff
@@ -1,33 +1,215 @@
+{{ end }}
+
 {{ define "header"}}
   <a class="sr-only" href="#content">Skip to main content</a>
   <header class="header-wrapper{{- with .Params.header_wrapper_class | default .Site.Params.header_wrapper_class }} {{ . }}{{end}}" id="header-wrapper">
-    {{ partial "navbar.html" . }}
-    
-    {{ if isset .Page.Params "headline" }}
-    <div class="jumbotron featured-jumbotron featured-jumbotron-download margin-bottom-0">
+    <!---Navbar--->
+    <div class="clearfix toolbar-container-wrapper">
       <div class="container">
-        <div class="row">
-          <div class="col-sm-14 col-sm-offset-3 text-left">
-            <h1>Jakarta&nbsp;EE <span class="hidden-sm hidden-md hidden-lg hidden-xl">8</span></h1>
-            {{ if isset .Page.Params "subtitle" }}
-            <h2>{{ htmlUnescape .Page.Params.subtitle }}</h2>
-            {{ end }} 
-             
-            <p class="text-center">An open, community-driven baseline for<br> the evolution of enterprise Java technologies</p>
-            {{ if isset .Page.Params "links" }}
-            <ul class="list-inline buttons">
-              {{range $key,$val := index .Page.Params.links}}
-                <li><a class="btn btn-primary" href="{{ (index $val 0).href}}">{{ (index $val 1).text}}</a></li>
-              {{end}}
+        <div class="text-right toolbar-row row hidden-print">
+          <div class="col-md-24 row-toolbar-col">
+            <ul class="list-inline">
+              <li><a href="https://accounts.eclipse.org/user/edit" class="" data-tab-destination="tab-profile"><i class="fa fa-edit fa-fw"></i> {{ i18n "navbar-edit-acccount-label" }}</a></li>
+              <li><a class="toolbar-link toolbar-manage-cookies dropdown-toggle"><i class="fa fa-wrench"></i> {{ i18n "navbar-manage-cookies-label" }}</a></li>
+              {{ if gt (len .AllTranslations) 1 }}
+              <li class="language-selector dropdown">
+                <a href="#" data-toggle="dropdown" class="dropdown-toggle">
+                  <i class="fa fa-globe"></i>
+                  <span> {{ .Language.LanguageName }}</span>
+                  <b class="caret"></b>
+                </a>
+                <ul class="dropdown-menu">
+                  {{ range .AllTranslations }}
+                  <li><a href="{{ .Permalink }}">{{ .Language.LanguageName }}</a></li>
+                  {{ end }}
+                </ul>
+              </li>
+              {{ end }}
             </ul>
-             {{end}} 
           </div>
-          <div class="col-xs-4 hidden-xs">
-            <span class="jumbotron-right-content">8</span>
+        </div>
+      </div>
+    </div>
+    <div class="{{ $.Site.Params.navbar_wrapper_classes | default "navbar-wrapper" }}">
+      <div class="container">
+        <div class="row" id="header-row">
+          <div class="{{ $.Site.Params.header_left_classes | default "col-sm-5 col-md-4" }}" id="header-left">
+            <div class="wrapper-logo-default">
```

Comment:
Oops, this is still using the white logo. 

@ericpoirier @autumnfound  Actually for this, since I just want to be able to add a colorful logo on this specific page, I'm thinking also make it customizable in https://github.com/EclipseFdn/hugo-solstice-theme

Reply:
https://github.com/EclipseFdn/hugo-solstice-theme/pull/179 added PR to customize logo